remote-trait-object/src/service/serde_support.rs

```diff
@@ -49,7 +59,14 @@ impl<T: ?Sized + Service> ServiceRef<T> {
 
     pub(crate) fn get_raw_export(self) -> ServiceToRegister {
         match self.service {
-            ExportOrImport::Export(x) => x,
+            ExportOrImport::Export(x) => {
+                let mut e = x.borrow_mut();
+                let old = std::mem::replace(&mut *e, ExportEntry::Exported(HandleToExchange(0)));
+                match old {
+                    ExportEntry::ReadyToExport(s) => s,
+                    _ => panic!(),
+                }
```

Comment:
How about using `RefCel::into_inner()`? Since get_raw_export's first argument is `self`, we can consume the `x`.
```suggestion
                 match x.into_inner() {
                     ExportEntry::ReadyToExport(s: ServiceToRegister) => s,
                     _ => panic!(),
                 }
```

Reply:
I forgot that it takes just `self`. Thanks.